acceptance/extension-logging-fluentd/src/__tests__/acceptance/fluent.acceptance.ts

```diff
@@ -50,7 +50,7 @@ describe('LoggingComponent', function () {
 
   /**
    * Read fluentd log files to match against the given regular exp
-   * @param regex A regular expression for assertion
+   * @param regex - A regular expression for assertion
```

Comment:
I'd prefer to use the short syntax ("optional hyphens" from https://github.com/microsoft/tsdoc/pull/206) and configure the linter to enforce this style everywhere.

```suggestion
   * @param regex A regular expression for assertion
```

However, it seems that `tsdoc` is preferring the hyphen-based flavor you have in your pull request, so I guess it's best to follow the idiomatic style and don't invent our own conventions 🤷

Reply:
Yeah, the plugin warns about the missing `-`.